FreeRTOSv202112.00/FreeRTOS/Demo/ARM7_LPC2129_Keil_RVDS/Task1 Files/source/task1.c

```diff
@@ -0,0 +1,22 @@
+#include "task1.h"
+#include "LED.h"
+// A task to toggle a LED every 1000 ms 
+void vToggleTask(void* pvParameters)
+{
+	for(;;)
+	{
+		led_toggle(1,1000);
```

Comment:
You shouldn't use magic numbers

Reply:
I resolved it

Reply:
https://github.com/MahmoudMatarawy/RTOS/blob/Task_1/FreeRTOSv202112.00/FreeRTOS/Demo/ARM7_LPC2129_Keil_RVDS/Task1%20Files/source/task1.c